tests/vmi_multus_test.go

```diff
@@ -664,6 +667,20 @@ var _ = Describe("SRIOV", func() {
 			return
 		}
 
+		createSriovVMI := func(networksNames []string) (vmi *v1.VirtualMachineInstance) {
+			SriovVMI := getSriovVmi(networksNames)
+
+			// This step is needed to guarantee that no VFs on the PF carry a duplicate MAC address that may affect
+			// ability of VMIs to send and receive ICMP packets on their ports.
```

Comment:
Can you please explain how can we get a duplicate mac address?

Reply:
Now I see the was a [comment](https://github.com/kubevirt/kubevirt/pull/3479/files#diff-897951be21489aebaa1df0c66894562cL840) explaining it pretty well, can you please move the whole deleted comment to here.